mockwebserver-deprecated/src/test/java/okhttp3/mockwebserver/MockWebServerTest.kt

```diff
@@ -500,11 +501,10 @@ class MockWebServerTest {
     val connection = url.openConnection() as HttpURLConnection
     assertThat(connection.getResponseCode()).isEqualTo(HttpURLConnection.HTTP_OK)
     val refusedConnection = url.openConnection() as HttpURLConnection
-    try {
+    assertFailsWith<ConnectException> {
       refusedConnection.getResponseCode()
-      fail<Any>("Second connection should be refused")
-    } catch (e: ConnectException) {
-      assertThat(e.message!!).contains("refused")
+    }.also { expected ->
```

Comment:
This idiom isn’t perfect, but it was easy enough to Structurally Replace and it makes the diffs pretty easy to review